app/services/update_client_vita_partner_service.rb

```diff
@@ -1,14 +1,21 @@
 class UpdateClientVitaPartnerService < BaseService
-  def initialize(clients:, vita_partner_id:, change_initiated_by: nil)
+  def initialize(clients:, vita_partner_id:, change_initiated_by: nil, first_time: false)
     @clients = clients
     @new_vita_partner = VitaPartner.find_by_id(vita_partner_id)
     @change_initiated_by = change_initiated_by
+    @first_time = first_time
```

Comment:
Are we using first_time anywhere in the code at the moment?

Reply:
Nope. I forgot to clean up a half-implemented idea. It's gone now. :)